### ISSUE TITLE
ARM: Failed to build: error[E0308]: mismatched types

### ISSUE THREAD

**Issue by jpentland** [ndesh26/evdev-rs#28]
When building on an arm device, I get:
```
   Compiling evdev-sys v0.2.0
   Compiling evdev-rs v0.3.0
error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/evdev-rs-0.3.0/src/lib.rs:166:19
    |
166 |         TimeVal { tv_sec, tv_usec }
    |                   ^^^^^^ expected i32, found i64
help: you can convert an `i64` to `i32` and panic if the converted value wouldn't fit
    |
166 |         TimeVal { tv_sec: tv_sec.try_into().unwrap(), tv_usec }
    |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0308]: mismatched types
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/evdev-rs-0.3.0/src/lib.rs:166:27
    |
166 |         TimeVal { tv_sec, tv_usec }
    |                           ^^^^^^^ expected i32, found i64
help: you can convert an `i64` to `i32` and panic if the converted value wouldn't fit
    |
166 |         TimeVal { tv_sec, tv_usec: tv_usec.try_into().unwrap() }
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
error: could not compile `evdev-rs`.

To learn more, run the command again with --verbose.
```

**Comment by ndesh26** [comment]
I think this error is fixed in the latest code with this commit https://github.com/ndesh26/evdev-rs/commit/6386065b4e49270818b1da653120a6f370c12352. If you can confirm that this commit fixes the error then I can release a new version with the fix.

**Comment by jpentland** [comment]
This is working for me on the same device.

**Comment by ndesh26** [comment]
I have published the version 0.3.1 which the has the above fix. Thanks for reporting the issue!!
